composer.json

```diff
@@ -28,32 +30,31 @@
     "source": "https://github.com/contao-community-alliance/events-contao-bindings"
   },
   "require": {
-    "php": ">=5.3",
-    "contao/core": ">=3.3,<3.6-dev",
-    "contao-community-alliance/composer-plugin": "~2.0",
-    "contao-community-alliance/event-dispatcher": "~1.0"
+    "php": ">=5.6",
+    "contao/core-bundle": "~4.4",
+    "contao-community-alliance/event-dispatcher": "~2.0@dev"
```

Comment:
I think, remove the cca event-dispatcher is right.
About add this `"symfony/event-dispatcher": "~3.0"`, what do you think?